modules/gltf/editor/editor_scene_importer_blend.h

```diff
@@ -64,6 +64,11 @@ class EditorSceneFormatImporterBlend : public EditorSceneFormatImporter {
 		BLEND_MODIFIERS_NONE,
 		BLEND_MODIFIERS_ALL
 	};
+	enum {
```

Comment:
I would prefer a named enum but this is the existing precedent so it's kinda out-of-scope of this PR.